style.css

```diff
@@ -26,265 +37,550 @@
-    justify-content: end;
-    width: 35px;
-    height: 3px;
-    /* background-color: #000; */
-    margin-top: 7px;
-    margin-right: 10px;
-    background-color: var(--primary_1);
-    font-size: 40px;
-  }
+  display: flex;
+  justify-content: end;
+  width: 35px;
+  height: 3px;
+  margin-top: 7px;
+  margin-right: 10px;
+  background-color: var(--primary_1);
+  font-size: 40px;
+}
 
 .sidebar {
   display: none;
 }
-  
-  .mobile-top-page {
-    background-image: url(./images/home-mobile.jpg);
-    background-repeat: no-repeat;
-    background-position: top center;
-    background-size: cover;
+
+.mobile-top-page {
+  background-image: url(./images/home-mobile.jpg);
+  background-repeat: no-repeat;
+  background-position: top center;
+  background-size: cover;
+}
+
+.mobile-first-page {
+  margin-right: 1em;
+  margin-left: 1em;
+  background-image: url(./images/bg-decorator-1.png), url(./images/bg-decorator-2.png), url(./images/bg-decorator-3.png), url(./images/bg-decorator-4.png);
+  background-repeat: no-repeat;
+  background-size: 20%, 12%, 5%, 5%;
+  background-position: 100% 50%, 60% 0%, 90% 3%, 5% 50%;
+}
+
+.mobile-first-page h1 {
+  font-family: 'Lato', sans-serif;
+  color: var(--primary_2);
+  font-size: 25px;
+  padding-top: 6%;
+  width: 100%;
+}
+
+.mobile-first-page h2 {
+  font-family: 'cocogoose', sans-serif;
+  color: transparent;
+  font-size: 40px;
+  width: 100%;
+  background-image: url(./images/font.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  -webkit-background-clip: text;
+}
+
+.mobile-first-page p {
+  margin-top: 20%;
+  background-color: var(--secondary_2);
+  border: 3px solid var(--secondary_1);
+  color: var(--primary_4);
+  padding: 20px;
+  line-height: 25px;
+  width: 100%;
+}
+
+.mobile-first-page h3 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 28px;
+  font-weight: 700;
+  color: var(--primary_4);
+  margin-top: 5%;
+}
+
+.mobile-first-page h4 {
+  font-family: 'Lato', sans-serif;
+  font-size: 23px;
+  color: var(--primary_4);
+  margin-top: 2%;
+  padding-bottom: 10%;
+}
+
+.mobile-second-page,
+.card-flex {
+  display: grid;
+  gap: 20px;
+  justify-content: center;
+  align-items: center;
+  background-image: url(./images/xpattern.jpg);
+  width: 100%;
+  background-repeat: repeat;
+  background-size: 10%;
+  background-color: var(--primary_3);
+  padding-top: 5%;
+}
+
+.mobile-second-page h2 {
+  font-family: 'Lato', sans-serif;
+  font-size: 25px;
+  font-weight: 800;
+  margin: 1em auto;
+  color: var(--secondary_1);
+}
+
+.mobile-second-page h2::after {
+  content: "";
+  background-color: var(--primary_2);
+  display: block;
+  margin: 0 auto;
+  width: 1.5em;
+  height: 2px;
+  border-radius: 0.5px;
+  margin-top: 0.5em;
+}
+
+.card-1,
+.card-2,
+.card-3,
+.card-4,
+.card-5 {
+  display: flex;
+  justify-content: space-between;
+  background-color: rgba(252, 241, 241, 0.277);
+  padding: 30px;
+  margin-left: 1em;
+  margin-right: 1em;
+  gap: 20px;
+}
```

Comment:
- [ ] To match the Behance design better, you should consider having your elements horizontally centered, as well as having set width for each element instead of space between, so they all text are aligned.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142471749-510f4ddb-429b-417a-aee9-afb374929f37.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/142471793-f4547c13-4a4e-432d-a449-33d131f214e6.png)